src/Model/Table/UsersTable.php

```diff
@@ -63,4 +64,26 @@ public function validationDefault(Validator $validator)
 
         return $validator;
     }
+
+    public function findMyStuffs(Query $query, array $options){
+        $userID = $options['userID'];
+
+//        //get username
+//        $users = TableRegistry::get("users");
+//        $Username = $users->find()
+//            ->where(['User1' => $userID]);
+
+        //get messages
+        $messages = TableRegistry::get('messages');
+        $msgToMe = $messages->find()
+            ->where(['User2' => $userID]);
```

Comment:
@raymondlf @bojms45 @jhuang28 after this line gets changed to the username, I think this will be good to go.